### DISCUSSION TITLE
Stopped working on adultdvdempire since a few days

### DISCUSSION THREAD

**Discussion by parmentlerr** [aclap-dev/video-downloadhelper#1182] [Help and Q&A]
### Your question

Hi,

I recently spotted that this wonderful addon has stopped working on adultdvdempire since a few days. Is it just for me or some global issue? A few days ago it was able to recognize played movie but now there is nothing at all :(.

### Addon & CoApp version and details

version: 9.1.0.43
target: google
channel: stable
lang: pl
coapp: {"found":true,"new_version":false,"path":"C:\\Program Files\\DownloadHelper CoApp\\vdhcoapp.exe","version":"2.0.19"}
license: {"unset":true}
platform: x86-64 win
UA: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/120.0.0.0 Safari/537.36 OPR/106.0.0.0
{
    "used_history_button": true,
    "show_success_notification_for_icognito": true,
    "view_options": {
        "all_tabs": true,
        "hide_downloaded": false,
        "low_quality": true,
        "show_button_clean": true,
        "show_button_clean_all": false,
        "show_button_convert_local": false,
        "sort_by_status": true,
        "sort_reverse": false
    },
    "use_legacy_ui": false,
    "use_sidebar": true,
    "open_count_store": 1
}

**Comment by GrampaWildWilly** [comment]
You need to supply at least one URL of a page on which something fails.  Otherwise, your problem can't be investigated.  If it's a paid item, there is an alternative.  See discussion #1146.
